oauth2_provider/constants.py

```diff
@@ -41,6 +41,10 @@
 SCOPE_NAME_DICT = dict([(name, value) for (value, name) in SCOPES])
 SCOPE_VALUE_DICT = dict([(value, name) for (value, name) in SCOPES])
 
+# since this package explicitly supports the 'id_token' response type, override
+# the base package's default.
+DEFAULT_RESPONSE_TYPE_CHOICES = ('code', 'token', 'id_token')
+RESPONSE_TYPE_CHOICES = getattr(settings, 'OAUTH_RESPONSE_TYPE_CHOICES', DEFAULT_RESPONSE_TYPE_CHOICES)
```

Comment:
on the one hand the base package's defaults are established [here](https://github.com/edx/django-oauth2-provider/blob/edx/provider/constants.py#L13) - since neither that package nor this one define their own runtime settings (instead it's done in the host django app, such as edx-platform) this seems like it should be effective.

on the other hand it doesn't work, at least in edx-platform.  it's still necessary to explicitly define the setting in platform, otherwise the base package's value defeats this one.  I think this is because the base package's constants end up being imported first.  and because of that, it almost seems not to matter what default is being set here.

i will fix this to append `id_token` as you'v'e suggested.

(btw is the correct term "overrided"?)